src/modules/TTS/TTSModule.ts

```diff
@@ -0,0 +1,291 @@
+
+                                        var startLineIndex = parseInt(splittingWord.style.getPropertyValue("--line-index"))
+
+                                        if (prevLineIndex >= 0 && prevLineIndex != startLineIndex) {
+                                            var prevLine = ".--line-index_" + prevLineIndex
+                                            let prevElements = node.querySelectorAll(prevLine);
+                                            var whitespaces: HTMLElement[] = []
+
+                                            prevElements.forEach(element => {
+                                                element.style.removeProperty("background")
+                                                var nextElement = element.nextElementSibling as HTMLElement
+                                                if (nextElement != null && nextElement.className.startsWith("whitespace")) {
+                                                    whitespaces.push(nextElement)
+                                                }
+                                            });
+
+
+                                            if (prevElements.length > 1) {                                                
+                                                whitespaces.forEach(element => {
+                                                    element.style.removeProperty("background")
+                                                });
+                                            }
+                                        }
+                                        
+                                        prevLineIndex = startLineIndex
+                                        var startLine = ".--line-index_" + startLineIndex
+                                        let elements = node.querySelectorAll(startLine);
+                                        var whitespaces1: HTMLElement[] = []
+                                        elements.forEach(element => {
+                                            element.style.background = self.tts.color
+                                            var nextElement = element.nextElementSibling as HTMLElement
+                                            if (nextElement != null && nextElement.className.startsWith("whitespace")) {
+                                                whitespaces1.push(nextElement)
+                                            }
+                                        });
+                                        if (elements.length > 1) {                                            
+                                            whitespaces1.forEach(element => {
+                                                element.style.background = self.tts.color
+                                            });
+                                        }
+
+
+                                    } else if (self.annotationModule.delegate.tts.highlight == "word") {
```

Comment:
Option names should be consistent - lines/words or line/word.

Reply:
this is actually from the splitting library, but i have to fix another issue with that library

Reply:
actually never mind, it's my mistake, it should be words